code/modules/mob/living/blood.dm

```diff
@@ -14,7 +14,7 @@
 	if(stat != DEAD && bodytemperature >= 170)	//Dead or cryosleep people do not pump the blood.
 		if(blood_volume > BLOOD_VOLUME_MAXIMUM) //Warning: contents under pressure.
 			var/spare_blood = blood_volume - ((BLOOD_VOLUME_MAXIMUM + BLOOD_VOLUME_NORMAL) / 2) //Knock you to the midpoint between max and normal to not spam.
-			if(drip(spare_blood))
+			if(drip(spare_blood, TRUE))
```

Comment:
I'd just cut the drip line entirely if you insist on this rather than adding a workaround. If(true) is weird, and there's no case where drip would return false here.

Reply:
Returning null is a falsey value, so there is a case.

Reply:
When would a null return be reached by this call? The TRUE prevents an early return, and if this line is reached blood volume is nonzero. NOBLOOD species aren't reaching it. In stasis, I guess? But then chems aren't processing, and I don't think there's any way to IV someone while they're also in a stasis effect.

Reply:
`.` is null by default in `drip`, so returning if theres qc in their system is returning null

Reply:
Which, like I said, is unreachable in this call because the TRUE argument prevents qc from mattering.

Reply:
oh that was your point sorry

Reply:
Drip function before changes:
```
/mob/living/carbon/proc/drip(amt)

	if(reagents.get_reagent_amount(/datum/reagent/medicine/quickclot)) //Quickclot stops bleeding, magic!
		return

	if(blood_volume)
		blood_volume = max(blood_volume - amt, 0)
		if(isturf(src.loc)) //Blood loss still happens in locker, floor stays clean
			if(amt >= 10)
				add_splatter_floor(loc)
			else
				add_splatter_floor(loc, 1)
		return TRUE
```
It's acknowledging the check for a 'blood_volume'.